sealedenumprocessor/src/main/kotlin/com/livefront/sealedenum/internal/SealedEnumTypeSpec.kt

```diff
@@ -16,42 +18,58 @@ import javax.lang.model.element.TypeElement
  * A builder for a [SealedEnum] object for the given [sealedClass].
  *
  * Given the [ClassName] of the [sealedClass], the [TypeElement] for the [sealedClassElement], and the
- * list of [ClassName]s for the sealed subclasses, [create] will generate a [TypeSpec] for an implementation of
+ * list of [ClassName]s for the sealed subclasses, [build] will generate a [TypeSpec] for an implementation of
  * [SealedEnum].
  *
- * If the sealed class is generic, pass in the number of type parameters as [numberOfTypeParameters]. The resulting
- * [SealedEnum] object will be the star-projections of the sealed class's type.
+ * Before calling [build], [addEnumForSealedEnumProvider] will additional let the generated object implement
```

Comment:
Should "additional" be "additionally" here?

Reply:
Fixed in https://github.com/livefront/sealed-enum/pull/10/commits/156ef92340d0f742a7a8b2a4ebbbe17e11e62baf